app/uk/gov/hmrc/customs/declarations/information/connectors/declarationsInformationConnectors.scala

```diff
@@ -92,6 +92,24 @@ class DeclarationSearchConnector @Inject()(http: HttpClient,
   override lazy val unavailablePeriodDurationInMillis = config.informationCircuitBreakerConfig.unavailablePeriodDurationInMillis
 }
 
+@Singleton
+class DeclarationFullConnector @Inject()(http: HttpClient,
```

Comment:
What are we going with? `FullDeclaration` or `DeclarationFull`?

Reply:
Connectors are named DeclarationVersionConnector, DeclarationSearchConnector, DeclarationFullConnector etc
It's FullDeclarationController that's a little out of step as controllers are SearchController, StatusController etc

Reply:
Ok, we'll come up with a consistent convention at the end.